.asf.yaml

```diff
@@ -30,6 +30,12 @@ github:
     rebase:  false
   protected_branches:
     master:
+      required_status_checks:
+        strict: true
+        contexts:
+          - license
+          - pecl
```

Comment:
Because you have a matrix for the job pecl, you have to include the matrix variable here, like `PECL (ubuntu-20.04, 8.1, 5.0.0)`, otherwise I’d recommend to add an aggregate job just like what you did for the test matrix. Or remove the matrix considering there is only one case in that matrix.

Reply:
Thx~